include/notifier.php

```diff
@@ -413,7 +418,7 @@ function notifier_run(&$argv, &$argc){
 			}
 
 			// Send a salmon to the parent owner
-			$r = q("SELECT `notify` FROM `contact` WHERE `nurl`='%s' AND `uid` IN (0, %d) AND `notify` != ''",
+			$r = q("SELECT `url`, `notify` FROM `contact` WHERE `nurl`='%s' AND `uid` IN (0, %d) AND `notify` != ''",
 				dbesc(normalise_link($thr_parent[0]['owner-link'])),
 				intval($uid));
 			if ($r)
```

Comment:
Standards: Please use `dbm::is_result()` and add braces to this conditional statement.